src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -59,22 +60,22 @@ internal static async Task<IEnumerable<FunctionMetadataResponse>> GetFunctionMet
             return await tasks.WhenAll();
         }
 
-        internal IEnumerable<FunctionMetadata> GetFunctionsMetadata(ScriptJobHostOptions hostOptions, ILogger logger, bool includeProxies = false)
+        internal IEnumerable<FunctionMetadata> GetFunctionsMetadata(bool includeProxies, bool forceRefresh)
         {
-            IEnumerable<FunctionMetadata> functionsMetadata = _functionMetadataProvider.GetFunctionMetadata();
+            Func<FunctionMetadata, bool> filterPredicate;
 
-            if (includeProxies)
+            if (!includeProxies)
             {
-                // get proxies metadata
-                var values = ProxyMetadataManager.ReadProxyMetadata(hostOptions.RootScriptPath, logger);
-                var proxyFunctionsMetadata = values.Item1;
-                if (proxyFunctionsMetadata?.Count > 0)
-                {
-                    functionsMetadata = proxyFunctionsMetadata.Concat(functionsMetadata);
-                }
+                // Remove all codeless metadata (includes proxies)
```

Comment:
So we're saying that all Codeless functions are always excluded when listing functions. We should include that detail in the public comment for the FunctionMetadata member.

Reply:
Yes, I did add it in the `<remarks></remarks>

https://github.com/Azure/azure-functions-host/pull/5657/files/9bfde909aff43c7c6e40f2a2e2dfc41ff3a0ba37#diff-7a0d8f754916c5e5775de01cbf573b4eR58